polkadot/zombienet_tests/elastic_scaling/0003-slot-based-3cores.zndsl

```diff
@@ -0,0 +1,19 @@
+Description: Test with slot-based collator using 3 cores and async backing
+Network: ./0003-slot-based-3cores.toml
+Creds: config
+
+elastic-validator: is up
+collator-elastic: is up
+collator-elastic-mvp: is up
+
+# Register 2 extra cores to each parachain.
+elastic-validator-0: js-script ./assign-core.js with "0,2100,57600" return is 0 within 600 seconds
+elastic-validator-1: js-script ./assign-core.js with "1,2100,57600" return is 0 within 600 seconds
+elastic-validator-2: js-script ./assign-core.js with "2,2200,57600" return is 0 within 600 seconds
+elastic-validator-3: js-script ./assign-core.js with "3,2200,57600" return is 0 within 600 seconds
+
+# Wait for 20 relay chain blocks 
+elastic-validator-0: reports substrate_block_height{status="best"} is at least 20 within 140 seconds
+
+elastic-validator-0: parachain 2100 block height is at least 20 within 30 seconds
```

Comment:
my general comment is that using zombienet-v1 for measuring throughput via timeout will always be hacky and result in flaky tests:
* we are waiting the substrate block height to be at least 20, but our intent is to wait for the next timeslice to kick in - this check might have false positives (block number > 20, but timeslice didn't kick in)
then we're trying to measure parachain throughput via block height and timeouts - again this is flaky, because we are not starting to wait from 0 - the risk is having "passing tests", but broken functionality

we should switch to using zombienet-sdk tests IMHO, especially for newly written tests. are there any blockers for doing this?